ml/src/test/scala/ai/h2o/sparkling/ml/algos/RegressionPredictionTestSuite.scala

```diff
@@ -58,21 +58,25 @@ class RegressionPredictionTestSuite extends FunSuite with Matchers with SharedH2
       .setWithDetailedPredictionCol(true)
       .setWithContributions(true)
       .setWithLeafNodeAssignments(true)
+      .setWithStageProbabilities(true)
       .setFeaturesCols("CAPSULE", "RACE", "DPROS", "DCAPS", "PSA", "VOL", "GLEASON")
       .setLabelCol("AGE")
 
     val model = algo.fit(dataset)
 
     val predictions = model.transform(dataset)
 
-    val expectedCols = Seq("value", "contributions", "leafNodeAssignments")
+    val expectedCols = Seq("value", "contributions", "leafNodeAssignments", "stageProbabilities")
     assert(predictions.select("detailed_prediction.*").schema.fields.map(_.name).sameElements(expectedCols))
     val contributions = predictions.select("detailed_prediction.contributions").head().getStruct(0)
     assert(contributions != null)
     assert(contributions.size == 8)
     val leafNodeAssignments = predictions.select("detailed_prediction.leafNodeAssignments").head().getSeq[String](0)
     assert(leafNodeAssignments != null)
     assert(leafNodeAssignments.length == algo.getNtrees())
+    val stageProbabilities = predictions.select("detailed_prediction.stageProbabilities").head().getList(0)
```

Comment:
I'm not sure whether makes sense to name the column `stageProbabilities` in case of regression problems.

Reply:
I'm thinking abou renaming `stageProbabilities` to `stagePredictions` for all problem types.